src/api-sdks/connection-api/clients/csharp/src/IdeaStatiCa.ConnectionApi/ConnectionApiServiceRunner.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Sockets;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Net.Http;
+
+namespace IdeaStatiCa.ConnectionApi
+{
+	public class ConnectionApiServiceRunner : IApiServiceFactory<IConnectionApiClient>
+	{
+		private const string LOCALHOST_URL = "http://127.0.0.1";
+		private const string API_EXECUTABLE_NAME = "IdeaStatiCa.ConnectionRestApi.exe";
+		private const string HEARTBEAT_ENDPOINT = "/heartbeat";
```

Comment:
RestApiConstants.RestApiHeartBeat already contains this. Please re-use that so we dont have constants on multiple places.